app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -2692,6 +2694,316 @@ class DatabaseMigrationAndroidTest {
     val dbV43 = helper.migrateTo(43)
     dbV43.assertTableDoesNotExist("Communication")
   }
+
+  @Test
+  fun migrating_ongoing_login_entry_to_44_should_add_the_new_columns() {
+    val db_v43 = helper.createDatabase(43)
+    val tableName = "OngoingLoginEntry"
+
+    db_v43.assertColumnCount(tableName, 3)
+
+    val db_v44 = helper.migrateTo(44)
+
+    db_v44.assertColumnCount(tableName, 9)
+  }
+
+  @Test
+  fun migrating_existing_ongoing_login_entry_to_44_should_migrate_exiting_data_from_not_logged_in_user() {
```

Comment:
TODO: Split this into multiple tests.

Reply:
Could not do this because it would make the tests too cumbersome. Rearranged the tests and verifications to be closer instead.